react-app-drawer-component/my-app/src/AppDrawer.js

```diff
@@ -0,0 +1,40 @@
+
+import { FaHamburger } from 'react-icons/fa';
+import './AppDrawer.css';
+
+/**
+ * AppDrawer is a hamburger menu component that toggles open and closed,
+ * and contains any number of pages.
+ * @param {heading} Represents name of menu.
+ * @param {items} An array containing pages listed within the menu.
+ */
+
+export default function AppDrawer({heading, items, onTitleChange, onMenuToggle, view}) {
```

Comment:
Instead of 'view' with a value of 'main', I probably would have just made it a boolean, `isOpen`.